### ISSUE TITLE
android:allowBackup="false"

### ISSUE THREAD

**Issue by lupajz** [yazeed44/MultiImagePicker#49]
I have android:allowBackup="false" set in my apps manifest uppon adding your dependecy I get : 

> Error:(6, 13) Gradle: Attribute application@allowBackup value=(false) from AndroidManifest.xml:6:13
> Error:(6, 13) Gradle: Execution failed for task ':app:processDebugManifest'.
> Manifest merger failed : Attribute application@allowBackup value=(false) from AndroidManifest.xml:6:13
  	is also present at net.yazeed44.imagepicker:imagepicker:1.1.6:12:9 value=(true)
  	Suggestion: add 'tools:replace="android:allowBackup"' to <application> element at AndroidManifest.xml:5:5 to override

**Comment by yazeed44** [comment]
>Suggestion: add 'tools:replace="android:allowBackup"' to element at AndroidManifest.xml:5:5 to override

Did you try that ?

**Comment by lupajz** [comment]
yes, as a temportal solution it seems to work, but I would prefer not having such tags in manifest

**Comment by yazeed44** [comment]
I will make a branch with ```android:allowBackup="false"``` and i want you to clone and try it  later
